code/modules/movespeed/modifiers/components.dm

```diff
@@ -7,6 +7,10 @@
 	multiplicative_slowdown = -7
 	movetypes = GROUND
 
+/datum/movespeed_modifier/snail_organ
```

Comment:
Rename this to something like "snail crawl" to be more specific.

Reply:
This is the innate snail species slowdown, I put it on the organ so that you can’t get the snails upsides without their downsides. There’s already a modifier called snail_crawl for when the snail is crawling

Reply:
I see.